### PR TITLE
Update USCMS-FNAL-WC1.yaml

### PR DESCRIPTION
**Pull request by hyunwoo18**
Adding a squid resource to the USCMS-FNAL-WC1

### CODE REVIEW

File: topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml [hunk 1]
```diff
@@ -405,5 +405,24 @@ Resources:
...
       StorageCapacityMax: 24000
       StorageCapacityMin: 24000
-      TapeCapacity: 0
+      TapeCapacity: 0      
+  cmsfrontier_squid:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml]:
```suggestion
          ID: e2799203e652f11e24de11fcf85b7e8d2b1783f7
```

File: topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml [hunk 2]
```diff
@@ -405,5 +405,24 @@ Resources:
...
+          Name: HYUNWOO KIM
+      Security Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml]:
```suggestion
          ID: e2799203e652f11e24de11fcf85b7e8d2b1783f7
```

File: topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml [hunk 3]
```diff
@@ -405,5 +405,24 @@ Resources:
...
+          Name: HYUNWOO KIM
+    Description: squid service for cms frontier
+    FQDN: cmsfrontiers.fnal.gov
+    ID:
```
Comment (brianhlin) [inline topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml]:
```suggestion
    ID: 1006
```

File: topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml [hunk 4]
```diff
@@ -405,5 +405,24 @@ Resources:
       KSI2KMin: 0
       StorageCapacityMax: 24000
       StorageCapacityMin: 24000
-      TapeCapacity: 0
+      TapeCapacity: 0      
+  cmsfrontier_squid:
```
Comment (brianhlin) [inline topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml]: Suggestion for the resource name to match the format of the other resources in the file:

```suggestion
  USCMS-FNAL-WC1-SQUID:
```

**Comment by hyunwoo18** [comment]
I am not so familiar with this github environment: How do I do this "bulk
commit"?
i.e. where in the page can I find an option of "bulk commit"?

**Comment by brianhlin** [comment]
Ah sorry, it's a "batch commit", not "bulk commit"

1. Go here https://github.com/opensciencegrid/topology/pull/655/files
1. Review each suggestion and if you approve, click "Add suggestion to batch". If you don't approve, please add a comment with your reasoning.
1. At the top of the page, click the "commit suggestions" button

**Comment by hyunwoo18** [comment]
Thanks!
It's done.
HyunWoo

**Pull request merged**